### ISSUE TITLE
Project Diva F 2nd - "Hold" notes are impossible with 60FPS enabled [PCSH00088]

### ISSUE THREAD

**Issue by ImplyingKinpatsu** [Electry/VitaGrafix#80]
Pressing a note often causes multiple presses and is guaranteed if you hold down the button, meaning "hold" notes will always fail

**Comment by SuperROBO-1** [comment]
I've noticed this with project diva F as well, only some songs though

**Comment by Yadalio** [comment]
Yes it really does! I thought if I touched it at first, but it just happened automatically...
